src/test/java/com/wikia/webdriver/common/dataprovider/MercurySEODataProvider.java

```diff
@@ -0,0 +1,48 @@
+package com.wikia.webdriver.common.dataprovider;
+
+import com.wikia.webdriver.common.core.url.Page;
+import org.testng.annotations.DataProvider;
+
+public class MercurySEODataProvider {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1118)